apps/web/public/static/locales/en/common.json

```diff
@@ -2471,7 +2471,6 @@
   "no_availability_shown_to_bookers": "If you don't assign anyone for this event, there will be no availability shown to bookers.",
   "go_back_and_assign": "Go back and Assign",
   "leave_without_assigning": "Leave without Assigning",
-  "signing_up_terms": "By proceeding, you agree to our <0>Terms</0> and <1>Privacy Policy</1>.",
```

Comment:
The values are different for those specifically the tags in them. Could you confirm we are deleting the right one?